pom.xml

```diff
@@ -234,10 +251,27 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>git-server</artifactId>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>4.2.1</version>
       <scope>test</scope>
     </dependency>
+    <!--
+      only to get the slf4j jul not here, the dependency should be provided in jenkins-war
+      plugins should be able to use what they prefer and not force to an implementation
+    -->
+    <dependency>
+      <groupId>org.jenkins-ci.main</groupId>
+      <artifactId>jenkins-war</artifactId>
+      <type>executable-war</type>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
because to have debug log for something such `org.apache.sshd.` it's so easier to have a simple `simplelogger.properties` file to use.
but it looks this has been removed.

Reply:
in your test code -> `new LoggerRule().capute("org.apache.sshd", Level.FINEST)`
you can also set a default log level for something for `hpi:run`  via [`loggers`](https://jenkinsci.github.io/maven-hpi-plugin/run-mojo.html#loggers)